apps/ledger-live-desktop/src/newArch/features/LNSUpsell/types/index.ts

```diff
@@ -3,12 +3,11 @@ import { Feature_LldNanoSUpsellBanners } from "@ledgerhq/types-live";
 export * from "./enum/Analytics";
 
 export type LNSBannerModel = {
-  location: LNSBannerLocation;
-  discount: number;
-  tracking: string;
-  image: string;
+  variant: "none" | "banner" | "notification";
```

Comment:
when there is more than 2 values, I'm not a fan of this type of typing, but it's ok

Reply:
totally agree